lib/modules/manager/terragrunt/modules.ts

```diff
@@ -10,6 +11,9 @@ import type { ExtractionResult, TerraformManagerData } from './types';
 export const githubRefMatchRegex = regEx(
   /github\.com([/:])(?<project>[^/]+\/[a-z0-9-_.]+).*\?(depth=\d+&)?ref=(?<tag>.*?)(&depth=\d+)?$/i,
 );
+export const gitlabRefMatchRegex = regEx(
```

Comment:
Here I'm not quite sure if this is the correct way since GitLab can be hosted on any (sub-)domain. Is there any possibility to cross-check with the configured registry URLs?

Reply:
Yes, better to use https://github.com/renovatebot/renovate/blob/45787c89a8a673954a065ee9670591dab94c45df/lib/util/common.ts#L12-L18

Reply:
Changed it to use the utility function. Now it also can support Bitbucket and Gitea as well.